static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -163,13 +164,19 @@ export const getUserEmailSignUps = (): Promise<any> => {
 };
 
 export const sendValidationEmail = (): any => {
+    const returnEndpoint = '/email-prefs';
     const endpoint = '/user/send-validation-email';
+    const returnUrl = getUrlVars().returnUrl
+        ? decodeURIComponent(getUrlVars().returnUrl)
+        : (idApiRoot || '') + returnEndpoint;
+
     const request = ajax({
         url: (idApiRoot || '') + endpoint,
         type: 'jsonp',
         crossOrigin: true,
         data: {
             method: 'post',
+            trackingReturnUrl: returnUrl,
```

Comment:
wary of calling this tracking as it's not for tracking.  Do we even use the tracking params once received?

Reply:
To be fair this is the only place I can see the tracking data used in identity https://github.com/guardian/identity/blob/master/identity-api/src/main/scala/com/gu/identity/api/social/SocialUserCreator.scala#L53